textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/KeywordPredicate.java

```diff
@@ -52,23 +59,48 @@ public KeywordPredicate(String query, List<Attribute> attributeList, Analyzer an
     }
 
     /**
-     * Creates a Query object as a boolean Query on all attributes.
+     * Creates a Query object as a boolean Query on all attributes
      * Example: For creating a query like
      * (TestConstants.DESCRIPTION + ":lin" + " AND " + TestConstants.LAST_NAME + ":lin")
      * we provide a list of AttributeFields (Description, Last_name) to search on and a query string (lin)
      *
      * TODO #88:BooleanQuery() is deprecated. In future a better solution could be worked out in Query builder layer
-
-     * @return QueryObject
+     *
+     * @return Query
      * @throws ParseException
      */
     private Query createQueryObject() throws ParseException {
+        ArrayList<String> tokens;
+        List<String> fieldList = new ArrayList<String>();
         BooleanQuery booleanQuery = new BooleanQuery();
```

Comment:
We want to make the dependency on Lucene more explicit in the code.  So rename `booleanQuery` to `lucene BooleanQuery`.